module2/homework/nwdNww/nwdNww.hpp

```diff
@@ -1,11 +1,29 @@
 #pragma once
+#include <cmath>
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    int mod;
+
+    if(lhs < 0) {
+        lhs = -lhs;
+    }
+    if(rhs < 0) {
+        rhs = -rhs;
+    }
```

Comment:
You can use `std::abs` instead of that conditions
```suggestion
    lhs = std::abs(lhs);
    rhs = std::abs(rhs);
```

Reply:
agree